package.json

```diff
@@ -54,7 +54,7 @@
     "test-gui": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js gui",
     "test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js modules",
     "test-dangerci-and-adminpage": "yarn danger ci && yarn test-adminpage",
-    "reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"_inc/client/**/*.{js,jsx}\""
+    "reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"_inc/client/**/*.{js,jsx}\" \"modules/**/*.{js,jsx}\""
```

Comment:
Should we also include js files in the `_inc` directory? We have a few JavaScript files in `_inc/lib` for example.

Reply:
Let's do it all!